proposals/0005-inproc-outofproc-compiler-library-support.md

```diff
@@ -2,7 +2,7 @@
 
 # Support for in-process, Out-of-process, or both for the c style compiler api
 
-* Proposal: [NNNN](NNNN-inproc-outofproc-compiler-api-support.md)
```

Comment:
Doh!  Just noticed that this file name doesn't match the actual name!.. incoming change!

Reply:
fixed!